djangocms_versioning_filer/monkeypatch/admin/clipboardadmin.py

```diff
@@ -26,6 +26,7 @@
 
 @csrf_exempt
 def ajax_upload(request, folder_id=None):
```

Comment:
I know this isn't code you just wrote, but potentially could be a security issue, so... Should this definitely be exempt from csrf checks? If so, why? It is trickier to set the csrf token in an ajax request, but certainly possible. See: https://docs.djangoproject.com/en/1.11/ref/csrf/#setting-the-token-on-the-ajax-request
Furthermore, shouldn't this have anykind of permission checks? From what I understand atm anybody can access this url. They don't even have to be logged in.
This is probably a separate issue, so I'm fine with ticketing this rather than solving it in this PR, but I do think it shouldn't be ignored as it could cause major security headaches in the future.

Reply:
Please ticket it then. Tx. @chaosk please could you comment on this also.